rb_ws/src/buggy/scripts/auton/autonsystem.py

```diff
@@ -252,7 +261,7 @@ def planner_tick(self):
             other_pose = self.get_world_pose(self.other_odom_msg)
 
         # update local trajectory via path planner
-        self.cur_traj, cur_idx = self.path_planner.compute_traj(
+        _, cur_idx = self.path_planner.compute_traj(
```

Comment:
if the goal was to decouple pathplanning and autonsystem, why are we returning cur_idx

Reply:
You are correct, I am about to push a commit that fixes this bug. I must've missed it in my first pass through.